server/services/wardrobe_gaps.py

```
@@ -0,0 +1,354 @@
+        _gap(
+            user_id=user_id,
+            severity="nice_to_have",
+            garment_category="Tops",
+            style_context=style,
+            reason=f"No neutral-color tops for {style} looks — adds palette flexibility.",
+            suggested_color="white",
+        )
+    ]
+
+
+def analyze_wardrobe_gaps(client: Client, user_id: str) -> list[dict[str, Any]]:
```

Ruff (0.15.12)

[warning] Too many branches (13 > 12)

(PLR0912)